src/markdown/tutorial/part-1/01-orientation.md

```diff
@@ -248,7 +167,7 @@ We'll learn about the purposes of these files and folders as we go. For now, jus
 
 Ember CLI comes with a lot of different commands for a variety of development tasks, such as the `ember new` command that we saw earlier. It also comes with a *development server*, which we can launch within the project with the `npm start` command:
 
-```run:server:start cwd=super-rentals expect="Serving on http://localhost:4200/"
+```run:server:start cwd=super-rentals expect="Local:   http://localhost:4200/"
```

Comment:
Are the docs linked below here about deploying still accurate?

Reply:
no, there are some very out of date things over there 🙈 time to add it to the list